### ISSUE TITLE
Template timesheet not found

### ISSUE THREAD

**Issue by saurabhnandu** [frappe/erpnext#25769]
## Description of the issue
There seems to be a timesheet template missing from the erpnext repo. Hence on Editing a Timesheet Detail item on the Timesheet page throws an error "Template timesheet not found"

## Context information (for bug reports)
When editing a timesheet every time an existing Timesheet Detail item is edited, an error is thrown. "Template timesheet not found"

The code below refers to the template timesheet, but this template is not found in the repo, hence the error is thrown.

[timesheet.js](https://github.com/frappe/erpnext/blob/develop/erpnext/projects/doctype/timesheet/timesheet.js)

```
time_logs_add: function(frm, cdt, cdn) {
		if(frm.doc.parent_project) {
			frappe.model.set_value(cdt, cdn, 'project', frm.doc.parent_project);
		}

		var $trigger_again = $('.form-grid').find('.grid-row').find('.btn-open-row');
		$trigger_again.on('click', () => {
			let $timer = $('.form-grid').find('[data-fieldname="timer"]');
			if ($timer.get(0)) {
				$timer.append(frappe.render_template("timesheet"));
			}
			frm.trigger("control_timer");
		});
	},
```

**Output of `bench version`**
```
bench_manager 0.0.1
erpnext 13.3.0
frappe 13.3.0
```

## Steps to reproduce the issue

1. Open an existing timesheet
2. Open an existing Timesheet Details child table entry

### Observed result
"Template timesheet not found" error is thrown. 

### Expected result
No error should be thrown. timesheet.html template should be present in the repo 

### Stacktrace / full error message

```
Template timesheet not found
```

## Additional information
Ubuntu 18, Easy Install Script.

**Comment by Alchez** [comment]
+1, we're seeing this issue as well

**Comment by Alchez** [comment]
This has been fixed with #25792.
